.github/workflows/main.yml

```diff
@@ -81,10 +81,14 @@ jobs:
             else
               PUSH="false"
             fi
+            
             VERSION=${TRAVIS_TAG}
             TAGS=$(echo $TAGS | sed 's!${{ env.DEFAULT_IMAGE}}!${{ env.DEV_IMAGE }}!')
             if [ "${{ matrix.node }}" == "${{ env.LATEST_NODE }}" ] && [ "${{ matrix.suffix}}" == "" ]; then
               TAGS="$TAGS,$IMAGE:$VERSION"
+              if [ "$TRAVIS_TAG" == *"beta"*  ]; then
+                TAGS="$TAGS,$IMAGE:beta"
```

Comment:
so what does this result in ? we used to have  beta, beta-10, beta-12, beta-10-minimal etc  *where just beta got you 10 I think - though now of course we should be 14

Reply:
This just adds a `beta` to match what would have been `latest` we can add more, just went for the easy one first after the comment on the forum